### ISSUE TITLE
`manylinux_2_39_aarch64` platform support

### ISSUE THREAD

**Issue by ZJEast** [facebookresearch/projectaria_tools#321]
My machine is DGX Spark.

```shell
  × No solution found when resolving dependencies:
  ╰─▶ Because all versions of projectaria-tools have no wheels with a matching platform tag (e.g., `manylinux_2_39_aarch64`)
      and vitra[visulization]==0.0.1 depends on projectaria-tools, we can conclude that vitra[visulization]==0.0.1 cannot
      be used.
      And because only vitra[visulization]==0.0.1 is available and you require vitra[visulization], we can conclude that your
      requirements are unsatisfiable.

      hint: Pre-releases are available for `projectaria-tools` in the requested range (e.g., 2.1.0a2), but pre-releases
      weren't enabled (try: `--prerelease=allow`)

      hint: Wheels are available for `projectaria-tools` (v2.1.1) on the following platforms: `manylinux_2_17_x86_64`,
      `manylinux2014_x86_64`, `macosx_11_0_arm64`
```

**Comment by ZJEast** [comment]
I want to reproduce "VITRA" (https://github.com/microsoft/VITRA) to see how it works in my robot. Looks like their data is based on Aria.
